cmd/upload.go

```diff
@@ -32,19 +53,35 @@ func init() {
 					fail(err)
 				}
 
-				scenarios := make([]string, len(args))
-				progressBar := progressbar.New(len(args) + 1)
-				progressBar.RenderBlank()
-
 				// TODO
 				// In the future, perhaps we do something a little more graceful than to
 				// use the git metadata from the last file uploaded. It seems we
 				// shouldn't ever be uploading files from multiple repositories under a
 				// single mapset.
 				var git *metadata.GitMetadata
 
-				for i, path := range args {
-					file, err := os.Open(path)
+				scenarioFiles := make([]string, 0, 10)
```

Comment:
Is there an advantage to using a smaller initial capacity? Seems like `len(args)` will be a better guess for the number of scenario files, at least most of the time.